src/test/rpc/AccountTx_test.cpp

```diff
@@ -244,6 +248,83 @@ class AccountTx_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testParametersApiV2()
+    {
+        using namespace test::jtx;
+        // test for API version 2
+        Env env{*this, envconfig([](std::unique_ptr<Config> c) {
+                    c->loadFromString("\n[beta_rpc_api]\n1\n");
+                    return c;
+                })};
```

Comment:
Since #4573 is merged, this can probably be changed to `Env env{*this};` ?

Reply:
Agreed.